### ISSUE TITLE
About replit

### ISSUE THREAD

**Issue by ryangolsing** [MYSeaIT/Sea-Search#1]
Hi, I just noticed that you posted the 200$ job for 20$. I was thinking about reporting you but it seemed little cruel. I think we can come to an agreement that would be beneficial for both of us. By the way if you don't response I will report you :)
https://replit.com/bounties/@CoderGautamYT/openai-whisper-short
https://replit.com/bounties/@SeaIT/assembly-ai-video-su

**Comment by MYSeaIT** [comment]
Yes what would you like dude, I already have it built in whisper but my man gautam wants it in assembly and anyways I'm going to give 90% of my work to the selected bounty hunter

**Comment by ryangolsing** [comment]
We know that is not true. If you open a job for 100$ and I get the bounty I won't report you.

**Comment by ryangolsing** [comment]
Ok, 5k is fine.

**Comment by MYSeaIT** [comment]
The whole process is on my replit. All the work I did and the help I got. 
Try another one.
I tried to explain myself to you which has only made the blackmailer come out of you. That is definitely not ethical.

**Comment by MYSeaIT** [comment]
However, if you are struggling and need work, I don't mind sending something your way when it comes up: Something within your area of expertise. 
Keep your chin up!

**Comment by ryangolsing** [comment]
It is not ethical to applying bunch of jobs with different backgrounds and not completing in time and  getting someone to do for you for a 10 times lower amount

**Comment by ryangolsing** [comment]
Don't worry I don't need money. I just don't like con artists.

**Comment by MYSeaIT** [comment]
Thus isn't going anywhere. 
You don't know anything about the procedures I follow and who I collaborate with. 
I am building a team and am also a part of other people's teams. 
Getting a part of your project done and paying a proportional amount is something you could also benefit from. It's called project management and outsourcing. If you get it done for a favourable amount, isn't that what all these platforms do?

**Comment by ryangolsing** [comment]
Ok, If you are innocent you don't have to worry about.

**Comment by MYSeaIT** [comment]
ok please message me on discord, you seem like a cool guy, id like to give one of my projects to you a whole project
